modules/react/pagination/stories/pagination.stories.mdx

```diff
@@ -139,20 +130,6 @@ return (
 );
 ```
 
-#### Props
```

Comment:
There's no prop table rendering (missing SymbolDoc?) but I could't figure out the right code to bring it back.

Reply:
Fixed